tests/test_biophysical/test_animal/test_feedlot/test_feedlot_ration.py

```
@@ -0,0 +1,144 @@
+        "feedlot_feeds": [301, 302],
+        "feedlot_starter_ration": {"301": 50, "302": 50},
+        "feedlot_transition_ration": {"301": 65, "302": 20, "303": 15},
+        "feedlot_finisher_ration": {"301": 70, "302": 20, "303": 10},
+    }
+    bad_config[f"feedlot_{phase}_ration"] = {"301": 40, "302": 40}  # sums to 80, not 100
+    with pytest.raises(ValueError, match=f"Feedlot {phase} ration percentages must sum to 100.0%"):
+        RationManager.set_ration_feeds(bad_config)
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize("bad_config,match", [
```

Ruff (0.15.17)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)